fcrepo-api-x-integration/README.md

```diff
@@ -0,0 +1,35 @@
+
+# API-X Integration tests
+
+To run integration tests, simply do 
+
+    mvn clean install
+    
+The integration tests use `cargo` to deploy an instance of Fedora.  After tests have completed, the Fedora logs can be found in
+
+    fcrepo-api-x-integration/target/cargo/configurations/jetty9x/logs/
+    
+You can also run the same Fedora instance with all test data intact by
+
+    cd fcrepo-api-x-integration
+    mvn cargo:run -Pstandard
+    
+This will run Fedora at port 8080
+
+## Relevant directories:
+* `target/fcrepo/` - Fedora's data store
+* `target/cargo/configurations/jetty9x/` - Jetty installation which runs fedora.  Contains logs in Jetty's standard location
+* `target/exam/{uuid}/` - Karaf installation directory
+
+## Pax Exam
+Pax exam runs integration tests are run in a Karaf container.  This has a few consequences and nuances:
+- Specific dependency jars (libraries, etc) are specified by _Karaf feature_ and retrieved from the local maven repository rather than the build reactor.  Karaf will load all dependencies listed in features files, and normal OSGi mechanisms will verify that
+all dependencies are satisfied.
+    - The set of OSGi dependencies may be different from non-osgi, which is why it's necessary to specifically test OSGi
+    - To add, remove, or update dependencies in a manner that is visible to Karaf, you have to track down the correct features file and edit it appropriately
+    - Karaf feature files are produced by artifacts in the project, it is necessary to include them in the pom like <pre>&lt;dependency&gt;
```

Comment:
Missing closing `</pre>`, which causes the format of the sub-bullet point to render and pre formatted text.